ts/proto.bzl

```diff
@@ -86,12 +86,12 @@ def ts_proto_library(name, node_modules, has_services = True, copy_files = True,
         directory_path(
             name = protoc_gen_connect_es_entry,
             tags = ["manual"],
-            directory = node_modules + "/@bufbuild/protoc-gen-connect-es/dir",
+            directory = node_modules + "/@connectrpc/protoc-gen-connect-es/dir",
```

Comment:
Isn't this a breaking change?

Reply:
yeah good catch, I'll have to add a parameter to the macro

Reply:
no actually, the doc at the top:

> **UNSTABLE API**: contents of this page are not subject to our usual semver guarantees.
> We may make breaking changes in any release.
> Please try this API and provide feedback.
> We intend to promote it to a stable API in a minor release, possibly as soon as v2.1.0.

Reply:
Since they made the change all the way back in last August I don't think it's worth the complexity of adding another parameter to the macro letting users go back. Anyhow it would be hard for them to discover it.